agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1922,9 +1951,11 @@ bool main_thread::hal_event_handler(bwl::base_wlan_hal::hal_event_ptr_t event_pt
             }
             roam_flag      = false;
             state_attempts = 0;
-            if (FSM_IS_IN_STATE(WIRELESS_ASSOCIATE_4ADDR_WAIT)) {
-                FSM_MOVE_STATE(WIRELESS_BRIDGE_DHCP);
-            }
+
+            // Send slaves to enable the AP's
+            send_slaves_enable();
+
+            FSM_MOVE_STATE(MASTER_DISCOVERY);
```

Comment:
No, it should be done as part of this commit, because we must go to discovery when enabling the APs.
The WIRELESS_ASSOICATE_4ADDR_WAIT condition was removed because we already check it before (see line 1905).